src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/gps/ElevatorPlate.java

```diff
@@ -119,45 +116,56 @@ public void openInterface(Player p, Block b) {
         if (floors.size() < 2) {
             SlimefunPlugin.getLocalization().sendMessage(p, "machines.ELEVATOR.no-destinations", true);
         } else {
-            openFloorSelector(b, floors, p);
+            openFloorSelector(b, floors, p, 1);
         }
     }
 
     @ParametersAreNonnullByDefault
-    private void openFloorSelector(Block b, List<Block> floors, Player p) {
-        CustomBookInterface book = new CustomBookInterface(SlimefunPlugin.instance());
-        ChatComponent page = null;
-
-        for (int i = 0; i < floors.size(); i++) {
-            if (i % 10 == 0) {
-                if (page != null) {
-                    book.addPage(page);
-                }
-
-                page = new ChatComponent(ChatColors.color(SlimefunPlugin.getLocalization().getMessage(p, "machines.ELEVATOR.pick-a-floor")) + "\n");
-            }
+    private void openFloorSelector(Block b, List<Block> floors, Player p, int page) {
```

Comment:
annotations?

Reply:
It has a `@ParametersAreNonnullByDefault` above